nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-cluster/src/main/java/org/apache/nifi/cluster/coordination/flow/PopularVoteFlowElection.java

```diff
@@ -141,7 +142,7 @@ public synchronized DataFlow castVote(final DataFlow candidate, final NodeIdenti
     }
 
     private String fingerprint(final DataFlow dataFlow) {
-        final String flowFingerprint = fingerprintFactory.createFingerprint(dataFlow.getFlow());
+        final String flowFingerprint = DigestUtils.sha256Hex(dataFlow.getFlow());
```

Comment:
With the change to flow fingerprinting, the `FingerprintFactory` no longer appears to be used in this class or elsewhere. Should the FingerprintFactory class be removed?  At minimum, it looks like the instance can be removed from this class.

Reply:
We can definitely remove the instance from this class. The class is still used elsewhere, though.